app/controllers/friend_requests_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FriendRequestsController < ApplicationController
+  before_action :set_friend_request, except: [:create]
+
+  def create
+    friend = User.find(params[:requested_id])
+    @friend_request = FriendRequest.new(requester_id: current_user.id, requested_id: friend.id)
+    if @friend_request.save
+      flash[:success] = 'Friend request has been sent'
+    else
+      flash[:danger] = 'Error sending friend request'
+    end
+    redirect_back(fallback_location: root_path)
+  end
+
+  def destroy
+    @friend_request.destroy
+  end
+
+  private
+
+  def set_friend_request
```

Comment:
Related to my comment above. Maybe we can just do away with declaring this separately since we're just using it in the `destroy` action. We can simply do the `@request = FriendRequest.find(params[:id])` in the `destroy` action